src/gateway/understanding-kong/key-concepts/routes.md

```diff
@@ -1,6 +1,24 @@
 ---
 title: Routes
-
+concept_type: explanation
 ---
 
-## PLACEHOLDER routes
+Routes determine how (and if) requests are sent to their services after they reach {{site.base_gateway}}. Where a service represents the backend API, a route defines what is exposed to clients. 
+
+A single service can have many routes. Once a route is matched, {{site.base_gateway}} proxies the request to its associated service.
+
+## Route and service interaction
+
+Routes, in conjunction with [services](/gateway/understanding-kong/key-concepts/routes/), let you expose your services to clients with {{site.base_gateway}}. {{site.base_gateway}} abstracts the service from the clients by using routes. Since the client always calls the route, changes to the services(like versioning) don't impact how clients make the call. Routes also allow the same service to be used by multiple clients and apply different policies based on the route used.
```

Comment:
```suggestion
Routes, in conjunction with [services](/gateway/understanding-kong/key-concepts/routes/), let you expose your services to clients with {{site.base_gateway}}. {{site.base_gateway}} abstracts the service from the clients by using routes. Since the client always uses the route to make a request, changes to the services, like versioning don't impact how clients make the request. Routes also allow the same service to be used by multiple clients and apply different policies based on the route used.
```